src/index.ts

```diff
@@ -315,6 +315,13 @@ export function offlineInit(config: IClientConfigSync): EppoClient {
   return EppoJSClient.instance;
 }
 
+type APIKey = string;
```

Comment:
nit: `SDKKey` is more accurate to avoid confusion

Reply:
done